### PR TITLE
Adding japanese channels

### LINKED ISSUE #5664: Space Shower TV
### Channel Name

Space Shower TV

### Channel ID

SpaceShowerTV.jp

### Country

Japan

### Language

Japanese

### Website

https://www.spaceshowertv.com/

### Stream URL

_No response_

### Notes

_No response_

### LINKED ISSUE #5665: Space Shower TV Plus
### Channel Name

Space Shower TV Plus

### Channel ID

SpaceShowerTVPlus.jp

### Country

Japan

### Language

Japanese

### Website

https://www.spaceshowertvplus.com/

### Stream URL

_No response_

### Notes

_No response_

### LINKED ISSUE #5666: Music On! TV (M-ON!)
### Channel Name

Music On! TV (M-ON!)

### Channel ID

MusicOnTV.jp

### Country

Japan

### Language

Japanese

### Website

https://www.m-on.jp/

### Stream URL

_No response_

### Notes

_No response_

### LINKED ISSUE #6709: Fuji TV ONE
### Channel Name

Fuji TV One Sports Variety

### Channel ID

FujiTVOne.jp

### Website

_No response_

### Stream URL

_No response_

### Notes

Fuji TV ONE Sports Variety is a japanese sports channel, who has the m3u of this channel, please tell me, thanks

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #6730: CS309 FuJi NEXT
### Channel Name

FuJi NEXT

### Channel ID

FujiTVNext.jp

### Website

_No response_

### Stream URL

_No response_

### Notes

This is a japanese channel, who has the m3u of this channel, please tell me, thanks

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #6755: Add: TBS Channel 1
### Channel Name

TBS Channel 1

### Channel ID

TBSChannel1.jp

### Website

https://www.tbs.co.jp/tbs-ch/tch1/

### Stream URL

No response

### Notes

This is a japanese channel, who has the m3u8 of this channel, please tell me, thanks

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #6840: Fuji TV Two
### Channel Name

Fuji TV Two

### Channel ID

FujiTVTwo.jp

### Website

_No response_

### Stream URL

_No response_

### Notes

This is a japanese channel, who has the m3u8 of this channel, please tell me, thanks

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7822: Add: MBS TV
### Channel Name

MBS TV

### Channel ID

JOOYDTV.jp

### Website

https://www.mbs.jp/

### Stream URL

_No response_

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9477: Add: Fuji TV
### Channel ID

JOCXDTV.jp

### Stream URL

_No response_

### Notes

Some Other Channel Requests

This Time Regarding Anime

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9482: Add: Kids Station
### Channel ID

KidsStation.jp

### Stream URL

_No response_

### Notes

Some Other Channel Requests

This Time Regarding Anime

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by kaikyoofers**
Links originally by @freearhey 
Closes https://github.com/iptv-org/iptv/issues/5664 Closes https://github.com/iptv-org/iptv/issues/5665 Closes https://github.com/iptv-org/iptv/issues/5666 Closes https://github.com/iptv-org/iptv/issues/6709 Closes https://github.com/iptv-org/iptv/issues/6730 Closes https://github.com/iptv-org/iptv/issues/6755 Closes https://github.com/iptv-org/iptv/issues/6840 Closes https://github.com/iptv-org/iptv/issues/7822 Closes https://github.com/iptv-org/iptv/issues/9477 Closes https://github.com/iptv-org/iptv/issues/9482 Closes https://github.com/iptv-org/iptv/issues/10807 Closes https://github.com/iptv-org/iptv/issues/11808 Closes https://github.com/iptv-org/iptv/issues/12352

### CODE REVIEW

**Review by freearhey — DISMISSED** [review]

File: streams/jp.m3u [hunk 1]
```diff
@@ -105,3 +105,57 @@ https://weather-live-hls01e.akamaized.net/ade36978-4ad3-48de-91ab-7d6edd0b6388/1
 https://nhkwlive-pre-in.webcdn.stream.ne.jp/hls/live/nhkwlive-pre/index_1M.m3u8
 #EXTINF:-1 tvg-id="NHKWorldJapan.jp",NHK World Japan
 https://nhkwlive-xjp.akamaized.net/hls/live/2003458/nhkwlive-xjp-en/index_1M.m3u8
+#EXTINF:-1 tvg-id="MusicOnTV.jp",Music On! TV
+http://rip.hidari.moe/get/jptv/get.php?id=6eHaPTJd0NoBEwN6E9jArw==
+#EXTINF:-1 tvg-id="MusicOnTV.jp",Music On! TV
+https://code.vthanhtivi.pw/getlink/jptv/_RVyS6Uym1zs93N3KvDcsw==/playlist.m3u8
+#EXTINF:-1 tvg-id="MusicOnTV.jp",Music On! TV
+https://stream01.willfonk.com/live_playlist.m3u8?cid=CS325&r=FHD&ccode=JP&m=d0:20:20:04:35:cc&t=0d6938cb3dcf4b79848bc1753a59daf1
```
Comment (BellezaEmporium) [inline streams/jp.m3u:113]: We were already told to not use these links before.

File: streams/jp.m3u [hunk 2]
```diff
@@ -105,3 +105,57 @@ https://weather-live-hls01e.akamaized.net/ade36978-4ad3-48de-91ab-7d6edd0b6388/1
...
+#EXTINF:-1 tvg-id="ATX.jp",AT-X
+https://code.vthanhtivi.pw/getlink/jptv/Nxk7xMmj7dV7iq_k08p2fQ==/playlist.m3u8
+#EXTINF:-1 tvg-id="ATX.jp",AT-X
+http://rip.hidari.moe/get/jptv/get.php?id=X4_3qVMPUBpZMqO_hX4vBw==
+#EXTINF:-1 tvg-id="KidsStation.jp",Kids Station
+https://code.vthanhtivi.pw/getlink/jptv/xuRJ2NXuC2ezP38iwgYdXQ==/playlist.m3u8
+#EXTINF:-1 tvg-id="KidsStation.jp",Kids Station
+http://rip.hidari.moe/get/jptv/get.php?id=xh4GJxN8zJFL3SmJ1P6Xsg==
+#EXTINF:-1 tvg-id="KidsStation.jp",Kids Station
+https://stream01.willfonk.com/live_playlist.m3u8?cid=CS330&r=FHD&ccode=JP&m=d0:20:20:04:35:cc&t=0d6938cb3dcf4b79848bc1753a59daf1
```
Comment (BellezaEmporium) [inline streams/jp.m3u:139]: Same here.

File: streams/jp.m3u (hunk 1 above)
Comment (kaikyoofers) [reply streams/jp.m3u:113]: Oh ok

**Comment by kaikyoofers** [comment]
@BellezaEmporium


Done

**Review by BellezaEmporium — APPROVED** [review]

**Comment by kaikyoofers** [comment]
also, why not use willfonk links?

**Comment by BellezaEmporium** [comment]
> also, why not use willfonk links?

They belong to a GitHub user that simply doesn't want his links to be used, as he pays for it.

**Comment by kaikyoofers** [comment]
@freearhey please review

**Review by freearhey — APPROVED** [review]

**Pull request merged**